crypto-keys/src/test/java/com/palantir/crypto2/keys/serialization/KeyMaterialsTest.java

```diff
@@ -109,6 +109,33 @@ public void testUnwrap_wrongVersion() {
                         KeySerializers.getSerializers().keySet());
     }
 
+    @Test
+    public void testSymmetric_wrapAndUnwrap() {
+        byte[] wrapped = KeyMaterials.symmetricWrap(keyMaterial, symmetricKey);
+        KeyMaterial unwrapped = KeyMaterials.symmetricUnwrap(wrapped, symmetricKey);
+        assertThat(unwrapped).isEqualTo(keyMaterial);
+    }
+
+    @Test
+    public void testSymmetric_unwrapInvalidWithWrongKey() {
+        SecretKey wrongKey = KeyMaterials.generateKey(KEY_ALG, KEY_SIZE);
+        byte[] wrapped = KeyMaterials.symmetricWrap(keyMaterial, symmetricKey);
+        KeyMaterial unwrapped = KeyMaterials.symmetricUnwrap(wrapped, wrongKey);
```

Comment:
with AES_GCM, I would expect this to throw because the MAC doesn't check out?

Reply:
done now that it's merged